src/pytest_qgis/pytest_qgis.py

```diff
@@ -322,12 +325,14 @@ def _show_qgis_dlg(common_settings: Settings, qgis_parent: QWidget) -> None:
     elif not common_settings.gui_enabled:
         warnings.warn(
             "Cannot show QGIS map because the GUI is not enabled. "
-            "Set qgis_qui_enabled=True in pytest.ini."
+            "Set qgis_qui_enabled=True in pytest.ini.",
+            stacklevel=1,
```

Comment:
is this needed since 1 is the default stacklevel?

Reply:
This is needed to please the "B028 No explicit stacklevel argument found." flake8 warning.